src/map/script.cpp

```diff
@@ -7855,6 +7856,11 @@ BUILDIN_FUNC(getitem2)
 					return SCRIPT_CMD_FAILURE;
 				}
 			}
+			int maxlen = 256;
+			char *buf = (char *)aMalloc(maxlen*sizeof(char));
```

Comment:
This gets leaked
 `sizeof(char)` is always 1, so you can just use maxlen
You don't need to cast the result of aMalloc() either.

Reply:
when i look to that now it's seen dumb, i can just use std::string :smile: